simplq/src/components/common/LoginButton.jsx

```diff
@@ -29,8 +30,7 @@ const LoginButton = () => {
   };
 
   if (loadingIndicator) {
-    // todo Use spinner
-    return <div>Loading...</div>;
+    return LoadingIndicator;
   }
```

Comment:
The onFailureCallback gets triggered in chrome on first visiting the site every time so there's this useless Login Failed error popup that shows up. It doesn't happen in firefox though. Trying to find a solution.

Reply:
I see. Report a bug.

 This loading indicator was not coming properly for me... it was coming somewhere in the middle of the screen. Now it's working?